eventmodel.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE "event" (
+  "event_id" integer PRIMARY KEY NOT NULL,
+  "title" varchar NOT NULL,
+  "tags" varchar,
+  "description" text,
+  "form_link" varchar NOT NULL,
+  "video_link" varchar NOT NULL,
+  "flyer" varchar,
```

Comment:
Let's assign a max length for each varchar, because `varchar` will assign it's default max length of 65535 which is huge for each attribute